docs/en/engines/table-engines/mergetree-family/mergetree.md

```diff
@@ -951,15 +955,20 @@ Optional parameters:
 -   `single_read_retries` — Number of retry attempts in case of connection drop during read. Default value is `4`.
 -   `min_bytes_for_seek` — Minimal number of bytes to use seek operation instead of sequential read. Default value is `1 Mb`.
 -   `metadata_path` — Path on local FS to store metadata files for S3. Default value is `/var/lib/clickhouse/disks/<disk_name>/`.
--   `cache_enabled` — Allows to cache mark and index files on local FS. Default value is `true`.
--   `cache_path` — Path on local FS where to store cached mark and index files. Default value is `/var/lib/clickhouse/disks/<disk_name>/cache/`.
 -   `skip_access_check` — If true, disk access checks will not be performed on disk start-up. Default value is `false`.
 -   `server_side_encryption_customer_key_base64` — If specified, required headers for accessing S3 objects with SSE-C encryption will be set.
 -   `s3_max_put_rps` — Maximum PUT requests per second rate before throttling. Default value is `0` (unlimited).
 -   `s3_max_put_burst` — Max number of requests that can be issued simultaneously before hitting request per second limit. By default (`0` value) equals to `s3_max_put_rps`.
 -   `s3_max_get_rps` — Maximum GET requests per second rate before throttling. Default value is `0` (unlimited).
 -   `s3_max_get_burst` — Max number of requests that can be issued simultaneously before hitting request per second limit. By default (`0` value) equals to `s3_max_get_rps`.
 
+Cache parameters:
```

Comment:
btw, there are much more cache parameters:

- max_size
- path
- max_elements
- max_file_segment_size
- cache_on_write_operations
- enable_filesystem_query_cache_limit
- enable_cache_hits_threshold
- enable_bypass_cache_with_threashold
- bypass_cache_threashold
- do_not_evict_index_and_mark_files

And probably better to mark that those are cache parameters and these
- disk
- type

are more like how cache layer is defined

Reply:
Thanks, I added notes to point to the full list.  I am going to move this information to an include file so that I can single-source it into the seven different places we have it so we can avoid having to remember to update it seven times.